std/base64.d

```diff
@@ -2060,7 +2060,8 @@ class Base64Exception : Exception
     // Verify that any existing workaround that uses & still works.
     InputRange ir2;
     OutputRange or2;
-    () @trusted {
+    () @trusted
+    {
```

Comment:
specifically I don't like this type of change

Reply:
this definitely looks more correct for allman style, would not remove this from D-Scanner but possibly add some kind of config for this

Reply:
That would be great, thanks!